src/test/java/com/google/enterprise/cloudsearch/fs/AclBuilderTest.java

```diff
@@ -325,6 +338,34 @@ public void testRemoveDomain() throws Exception {
     assertEquals(expected, result);
   }
 
+  @Test
+  public void getAcl_externalGroups_mapped() throws Exception {
+    File groupsFile = temporaryFolder.newFile();
+    // Group members aren't used when mapping external groups to identity sources.
```

Comment:
It took me a while to get this. Maybe "Group members aren't used in this context. The connector only maps external groups to their identity sources for use in ACLs."